apps/passport-server/src/services/issuanceService.ts

```diff
@@ -887,14 +1067,133 @@ export class IssuanceService {
       return tickets;
     });
   }
+
+  /**
+   * Verifies a ticket based on:
+   * 1) verification of the PCD (that it is correctly formed, with a proof
+   *    matching the claim)
+   * 2) whether the ticket matches the ticket types known to us, e.g. Zuzalu
+   *    or Zuconnect tickets
+   *
+   * Not used for Devconnect tickets, which have a separate check-in flow.
```

Comment:
worth noting what tickets it IS used for, more explicity.
```suggestion
   * Not used for Devconnect tickets, which have a separate check-in flow.
   * Used ONLY for Zuconnect '23 and Zuzalu '23 tickets
```

Reply:
It is used for all tickets other than Devconnect: for unknown tickets the response is `{ verified: true }` and for known tickets the response is something like `{ verified: true, knownTicketType: true, publicKeyName: "xxxx", group: KnownTicketGroup.Zuzalu23 }`.